internal/controller/management/projectconfigs/project_configs.go

```diff
@@ -277,7 +276,7 @@ func (r *reconciler) newWebhookReceiver(
 			string(secret),
 		),
 	}
-	wr.URL = path.Join(r.cfg.ExternalWebhookServerBaseURL, wr.Path)
+	wr.URL = r.cfg.ExternalWebhookServerBaseURL + wr.Path
```

Comment:
I think url.JoinPath() might cover some edge cases better than concatenation would. tbh, I didn't realize url.JoinPath() existed until just now. There are probably many more spots where we could have taken advantage of that.